Ooui.Wasm.Build.Tasks/BuildDistTask.cs

```diff
@@ -16,7 +16,7 @@ namespace Ooui.Wasm.Build.Tasks
 {
     public class BuildDistTask : Task
     {
-        const string SdkUrl = "https://xamjenkinsartifact.azureedge.net/test-mono-mainline-webassembly/108/highsierra/sdks/wasm/mono-wasm-a14f41ca260.zip";
+        const string SdkUrl = "https://jenkins.mono-project.com/job/test-mono-mainline-wasm/label=ubuntu-1804-amd64/916/Azure/processDownloadRequest/916/ubuntu-1804-amd64/sdks/wasm/mono-wasm-f25f9e5f2b5.zip";
```

Comment:
The mono team does not want us using those URLs, see: https://github.com/praeclarum/Ooui/commit/e94436c987f85ad9e142cf374869e9b88a11241d#diff-aa76d99f1a396cf6c560ff6b3c5b3b62

We should only pull from `xamjenkinsartifact.azureedge.net` and it should be as simple as changing the build number from `108` to `916` or whatever they're up to.